CHANGELOG.md

```diff
@@ -1,3 +1,47 @@
+### citus v11.1.6 (April 20, 2023) ###
```

Comment:
Apparently we missed adding an entry for 2) in 11.2, probably my bad. I think it's ok to use the description noted in PR for 11.1.x release.

Reply:
For 3): "Fix compilation for PG13.10 and PG14.7"

Reply:
@JelteF do you think it is ok if we do not need to backport this fix for compilation issues to earlier versions of Citus?

Citus 9.5.0 added support for PG13 and it is possible that we need the same fix to a version as early as this one.

Reply:
It shouldn't be necessary for any citus version that doesn't support PG15, because it fixes something in our PG15 support to also work for new patch versions of PG13/PG14